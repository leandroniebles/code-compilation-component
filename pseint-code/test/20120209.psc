proceso test

  num<-50;
  escribir num>=10 & num<=100;
  escribir (num>=10) & (num<=100);
  escribir (num>=10 & num<=100);
  escribir ((num>=10) & (num<=100));
  escribir (((num>=10)) & ((num<=100)));
  escribir ((num>=10) &(num<=100));
  escribir ((num>=10)& (num<=100));
  escribir num>=10 &num<=100;
  escribir num>=10&num<=100;
  escribir num>=10& num<=100;
  escribir (num>=10)&(num<=100);
  escribir (num>=10) &(num<=100);
  escribir (num>=10)& (num<=100);
  escribir ((num>=10)&(num<=100));
  escribir ((num>=10) &(num<=100));
  escribir ((num>=10)& (num<=100));

  escribir "";

  num<-5;
  escribir num>=10 & num<=100;
  escribir (num>=10) & (num<=100);
  escribir (num>=10 & num<=100);
  escribir ((num>=10) & (num<=100));
  escribir (((num>=10)) & ((num<=100)));
  escribir ((num>=10) &(num<=100));
  escribir ((num>=10)& (num<=100));
  escribir num>=10 &num<=100;
  escribir num>=10&num<=100;
  escribir num>=10& num<=100;
  escribir (num>=10)&(num<=100);
  escribir (num>=10) &(num<=100);
  escribir (num>=10)& (num<=100);
  escribir ((num>=10)&(num<=100));
  escribir ((num>=10) &(num<=100));
  escribir ((num>=10)& (num<=100));

  escribir "";

  num<-105;
  escribir num>=10 & num<=100;
  escribir (num>=10) & (num<=100);
  escribir (num>=10 & num<=100);
  escribir ((num>=10) & (num<=100));
  escribir (((num>=10)) & ((num<=100)));
  escribir ((num>=10) &(num<=100));
  escribir ((num>=10)& (num<=100));
  escribir num>=10 &num<=100;
  escribir num>=10&num<=100;
  escribir num>=10& num<=100;
  escribir (num>=10)&(num<=100);
  escribir (num>=10) &(num<=100);
  escribir (num>=10)& (num<=100);
  escribir ((num>=10)&(num<=100));
  escribir ((num>=10) &(num<=100));
  escribir ((num>=10)& (num<=100));

  escribir "";
  escribir "";

  num<-50;
  escribir num<10 | num>100;
  escribir (num<10) | (num>100);
  escribir (num<10 | num>100);
  escribir ((num<10) | (num>100));
  escribir (((num<10)) | ((num>100)));
  escribir ((num<10) |(num>100));
  escribir ((num<10)| (num>100));
  escribir num<10 |num>100;
  escribir num<10|num>100;
  escribir num<10| num>100;
  escribir (num<10)|(num>100);
  escribir (num<10) |(num>100);
  escribir (num<10)| (num>100);
  escribir ((num<10)|(num>100));
  escribir ((num<10) |(num>100));
  escribir ((num<10)| (num>100));

  escribir "";

  num<-5;
  escribir num<10 | num>100;
  escribir (num<10) | (num>100);
  escribir (num<10 | num>100);
  escribir ((num<10) | (num>100));
  escribir (((num<10)) | ((num>100)));
  escribir ((num<10) |(num>100));
  escribir ((num<10)| (num>100));
  escribir num<10 |num>100;
  escribir num<10|num>100;
  escribir num<10| num>100;
  escribir (num<10)|(num>100);
  escribir (num<10) |(num>100);
  escribir (num<10)| (num>100);
  escribir ((num<10)|(num>100));
  escribir ((num<10) |(num>100));
  escribir ((num<10)| (num>100));

  escribir "";

  num<-105;
  escribir num<10 | num>100;
  escribir (num<10) | (num>100);
  escribir (num<10 | num>100);
  escribir ((num<10) | (num>100));
  escribir (((num<10)) | ((num>100)));
  escribir ((num<10) |(num>100));
  escribir ((num<10)| (num>100));
  escribir num<10 |num>100;
  escribir num<10|num>100;
  escribir num<10| num>100;
  escribir (num<10)|(num>100);
  escribir (num<10) |(num>100);
  escribir (num<10)| (num>100);
  escribir ((num<10)|(num>100));
  escribir ((num<10) |(num>100));
  escribir ((num<10)| (num>100));

finproceso