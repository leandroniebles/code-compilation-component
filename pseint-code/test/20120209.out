VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO

FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO

FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO


FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO
FALSO

VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO

VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
VERDADERO
